examples/lcls/variables.yml

```
@@ -1,29 +1,29 @@
 input_variables:
   klys_li24_11:
-    type: scalar
-    default: 1 # will be dropped in later lume-epics iter
+    variable_class: ScalarVariable
+    default_value: 1 # will be dropped in later lume-epics iter
     range: [0, 100] # will be dropped in later lume-epics iter
```

Same for all variables below
```suggestion
    value_range: [0, 100] # will be dropped in later lume-epics iter
```